### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by filip-michalsky**
fix docs links to the documentation

adding badges to increase curb appeal

after merge - I will create 1.2.1 release.

### CODE REVIEW

**Review by skadio — APPROVED** [review]

**Comment by bkleyn** [comment]
I know there's no code changes, but probably still makes sense to update `_version.py` and `CHANGELOG.txt` to reflect changes and be consistent with PyPI version?

File: jurity/_version.py [hunk 1]
```diff
@@ -2,4 +2,4 @@
 # Copyright FMR LLC <opensource@fidelity.com>
 # SPDX-License-Identifier: Apache-2.0
 
-__version__ = "1.2.0"
+__version__ = "1.2.1"
```
Comment (dorukkilitcioglu) [inline jurity/_version.py:5]: Can you update the version in the docs as well? https://github.com/fidelity/jurity/blob/master/docsrc/conf.py

**Review by dorukkilitcioglu — APPROVED** [review]
Needs a minor version change in the docs, but looks good otherwise.

**Comment by filip-michalsky** [comment]
> I know there's no code changes, but probably still makes sense to update `_version.py` and `CHANGELOG.txt` to reflect changes and be consistent with PyPI version?

`CHANGELOG.txt` and `_version.py` both updated.

**Pull request merged**